e2e_tests/tests/command/test_tensorboard.py

```diff
@@ -159,3 +160,44 @@ def test_start_tensorboard_for_multi_experiment(tmp_path: Path, secrets: Dict[st
                 break
         else:
             raise AssertionError(f"Did not find {SERVICE_READY} in output")
+
+
+@pytest.mark.slow
+@pytest.mark.e2e_gpu
+@pytest.mark.tensorflow2
+@pytest.mark.parametrize("prefix", [None, "my/test/prefix"])
+def test_start_tensorboard_with_custom_image(
+    tmp_path: Path, secrets: Dict[str, str], prefix: Optional[str]
+) -> None:
+    """
+    Start a random experiment configured with the shared_fs backend, start a
+    TensorBoard instance pointed to the experiment, and kill the TensorBoard
+    instance.
+    """
+    with FileTree(tmp_path, {"config.yaml": s3_config(1, secrets, prefix)}) as tree:
```

Comment:
can you allow the default configuration to take over, rather than force an s3 backend?  This is the sort of e2e test which should not depend on cloud credentials.